File: program107.c
/*input  row=4   column =4

output  
* 
A 
B B 
C C C 
D D D D

complexity 
*/
#include<stdio.h>
void Display(int iRow,int iCol)
{
    int i=0,j=0;
    char ch='\0';
    if(iRow!=iCol)
{
    printf("Invalid input\n");
    return;
}

    for(i=1,ch='A';i<=iRow;i++,ch++)
    {
        for(j=1;j<=i;j++)
        {
            printf("%c\t",ch);
        }
         printf("\n");
    }
}
int main()
{
    int iValue1=0,iValue2=0;

    printf("Please enter number of rows\n");
    scanf("%d",&iValue1);
    printf("Please enter number of columns\n");
    scanf("%d",&iValue2);

    Display(iValue1,iValue2);

    return 0;
}

File: program107.java
/*input  row=4   column =4

output  
* 
A 
B B 
C C C 
D D D D

*/


import java.lang.*;
import java.util.*;

class Pattern
{
    public void Display(int row,int col)
    {
        int i=0,j=0;
        char ch='\0';
        if(row!=col)
        {
            System.out.println("Invalid input");
            return;
        }

        for(i=1,ch='A';i<=row;i++,ch++)
        {
            for(j=1;j<=i;j++)
            {
                {
                    System.out.print(ch+"\t");
                }
            }
            System.out.println();
        }

    }
}

class program107
{
    public static void main(String arg[])
    {

        Scanner sobj=new Scanner(System.in);
        Pattern obj=new Pattern();
        System.out.println("Enter number of rows");
        int no1=sobj.nextInt();
        System.out.println("Enter number of cols");
        int no2=sobj.nextInt();

        obj.Display(no1,no2);


    }
}